x-pack/plugins/maps/public/connected_components/mb_map/tile_status_tracker/tile_status_tracker.tsx

```diff
@@ -207,6 +217,47 @@ export class TileStatusTracker {
     }
   }, 100);
 
+  _updateTileStatusForLayer = (layer: ILayer, areTilesLoaded: boolean, errorMessage?: string) => {
+    this.props.setAreTilesLoaded(layer.getId(), areTilesLoaded);
+
+    if (errorMessage) {
+      this.props.setTileLoadError(layer.getId(), errorMessage);
+    } else {
+      this.props.clearTileLoadError(layer.getId());
+    }
+
+    const source = layer.getSource();
+    if (
+      layer.isVisible() &&
+      source.isESSource() &&
+      typeof (source as IVectorSource).isMvt === 'function' &&
+      (source as IVectorSource).isMvt()
+    ) {
+      // querySourceFeatures can return duplicated features when features cross tile boundaries.
+      // Tile meta will never have duplicated features since by there nature, tile meta is a feature contained within a single tile
```

Comment:
```suggestion
      // Tile meta will never have duplicated features since by their nature, tile meta is a feature contained within a single tile
```